go/chat/storage/outbox.go

```diff
@@ -522,19 +531,26 @@ func (o *Outbox) EphemeralPurge(ctx context.Context) (err error) {
 
 	obox.Records = recs
 
-	// Write out box
+	// Write out diskbox
 	if err := o.writeDiskBox(ctx, o.dbKey(), obox); err != nil {
 		return o.maybeNuke(NewInternalError(ctx, o.DebugLabeler,
 			"error writing outbox: err: %s", err.Error()), o.dbKey())
 	}
 
-	if len(purged) > 0 {
+	if len(ephemeralPurged) > 0 {
 		act := chat1.NewChatActivityWithFailedMessage(chat1.FailedMessageInfo{
-			OutboxRecords:    purged,
+			OutboxRecords:    ephemeralPurged,
 			IsEphemeralPurge: true,
 		})
 		o.G().NotifyRouter.HandleNewChatActivity(context.Background(),
 			keybase1.UID(o.GetUID().String()), &act)
 	}
+	if len(purged) > 0 {
```

Comment:
I don't think you need this, the main goal here is to just get these things out of the box, not get them off the screen, that can just happen whenever.

Reply:
makes sense, we just need it for ephemeral stuff